lib/snippets/push-to-heroku

```diff
@@ -62,11 +62,12 @@ if [ "$MIGRATIONS" != "" ]; then
     exit 1
   fi
 
-  green "Post-migration dyno restart."
-  heroku restart --app $HEROKU_APP
+  if [ -z $NO_POST_MIGRATION_RESTART ]; then
```

Comment:
I like avoiding negated variables -- go with `if [[ ${POST_MIGRATION_DEPLOY} ]]` with a default truthy value set for `POST_MIGRATION_DEPLOY` in line 16